boards/nxp/frdm_ke15z/frdm_ke15z.dts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_ke1xz.dtsi>
+#include "frdm_ke15z-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "NXP Freedom KE15Z board";
+	compatible = "nxp,ke15z", "nxp,mke15z7";
+
+	aliases {
+		led0 = &green_led;
+		led1 = &blue_led;
+		led2 = &red_led;
+		sw0 = &user_button_0;
+		sw1 = &user_button_1;
+	};
+
+	chosen {
+		zephyr,sram = &sram_u;
+		zephyr,flash = &flash0;
+		zephyr,console = &lpuart1;
+		zephyr,shell-uart = &lpuart1;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		red_led: led_0 {
+			gpios = <&gpiod 0 GPIO_ACTIVE_LOW>;
+			label = "RED LED";
+		};
+		green_led: led_1 {
+			gpios = <&gpiod 16 GPIO_ACTIVE_LOW>;
+			label = "GREEN LED";
+		};
+		blue_led: led_2 {
+			gpios = <&gpiod 15 GPIO_ACTIVE_LOW>;
+			label = "BLUE LED";
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		user_button_0: button_0 {
+			label = "User SW3";
+			gpios = <&gpiod 3 GPIO_ACTIVE_LOW>;
+			zephyr,code = <INPUT_KEY_0>;
+		};
+		user_button_1: button_1 {
+			label = "User SW2";
+			gpios = <&gpiob 11 GPIO_ACTIVE_LOW>;
+			zephyr,code = <INPUT_KEY_1>;
+		};
+	};
+};
+
+&cpu0 {
+	clock-frequency = <48000000>;
```

Comment:
Why set this at board level? The clock settings to achieve this core frequency are all done at the SOC level